tests/Sdk/Unit/Metrics/MeterTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Metrics;
+
+use OpenTelemetry\Sdk\Metrics\Counter;
+use OpenTelemetry\Sdk\Metrics\Meter;
+use OpenTelemetry\Sdk\Metrics\UpDownCounter;
+use OpenTelemetry\Sdk\Metrics\ValueRecorder;
+use PHPUnit\Framework\TestCase;
+
+class MeterTest extends TestCase
+{
+    public function testMeterInstrumentCreation()
+    {
+        $meter = new Meter('Meter', '0.1');
+        $this->assertInstanceOf(Meter::class, $meter);
+
+        $meter_name = $meter->getName();
+        $this->assertSame('Meter', $meter_name);
```

Comment:
well actually I don't think you need `$meter_name` variable at all. IMHO it's more readable to call `getName()` right in the assertion so you see exaclty what is being compared

Reply:
Yes, that's a bit cleaner!